assets/javascripts/discourse/initializers/chat-keyboard-shortcuts.js

```diff
@@ -47,6 +47,8 @@ export default {
       }
       return true;
     };
+    const isDrawerExpanded = () =>
+      document.querySelector(".topic-chat-container.expanded").length === 1;
```

Comment:
Can we add the brackets around this so it looks more clear?

Also, I'm getting an error here:
```
plugins/discourse-chat/assets/javascripts/javascripts/discourse/initializers/chat-keyboard-shortcuts:60 Uncaught TypeError: Cannot read properties of null (reading 'length')
```